plugins/agents/muninpluginrunner/exec.go

```diff
@@ -0,0 +1,74 @@
+package muninpluginrunner
+
+import (
+	"bufio"
+	"regexp"
+	"strconv"
+
+	"github.com/abrander/agento/plugins"
+	"github.com/abrander/agento/timeseries"
+)
+
+func init() {
+	plugins.Register("muninpluginrunner", newMuninPluginRunner)
+}
+
+// MuninPluginRunner will retrieve stub status.
+type MuninPluginRunner struct {
+	Cmd string `toml:"cmd" json:"cmd" description:"Command to run"`
+	Arg string `toml:"arg" json:"arg" description:"Arguments to command"`
+	kv  []KeyValue
+}
+
+type KeyValue struct {
+	key   string
+	value float64
+}
+
+func newMuninPluginRunner() interface{} {
+	return new(MuninPluginRunner)
+}
+
+// Gather will execute command (with arguments) and read each line in output.
+// Gather expect output to be munin plugin style:
+// http://munin-monitoring.org/wiki/HowToWritePlugins
+func (m *MuninPluginRunner) Gather(transport plugins.Transport) error {
+	stdout, _, _ := transport.Exec(m.Cmd, m.Arg)
```

Comment:
It would be nice to do some form of error checking here. Now the user have no idea if the script failed or simply didn't produce any output.

Reply:
@abrander err is now returned